### PR TITLE
Implement GA using celery

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony Please review

File: babycenter/settings/base.py [hunk 1]
```diff
@@ -452,3 +455,30 @@
...
+]
+
+CUSTOM_GOOGLE_ANALYTICS_IGNORE_PATH = environ.get(
+    'GOOGLE_ANALYTICS_IGNORE_PATH')
+if CUSTOM_GOOGLE_ANALYTICS_IGNORE_PATH:
+    GOOGLE_ANALYTICS_IGNORE_PATH += [
+        d.strip() for d in CUSTOM_GOOGLE_ANALYTICS_IGNORE_PATH.split(',')]
+
+
+CSRF_FAILURE_VIEW = 'molo.core.views.csrf_failure'
```
Comment (miltontony) [inline babycenter/settings/base.py:481]: You'll need to bump the version of `molo.core` in `requirements.txt` to the latest.

**Comment by miltontony** [comment]
1 comment, otherwise 👍

**Comment by smarzban** [comment]
👍

**Pull request merged**